lib/components/Icon/Icon.tsx

```diff
@@ -71,33 +71,25 @@ export interface IconProps extends React.Props<IconType> {
  *
  * @param props Control properties (Defined in `IconProps` interface)
  */
-export const Icon: React.StatelessComponent<IconProps> = (props: IconProps) => {
-    let iconClassName = `icon-${props.icon}`;
-    let cls = css({
+export const Icon = React.memo((props: IconProps) => {
+    const iconClassName = `icon-${props.icon}`;
+    const cls = css({
         'icon-xsmall': props.size === IconSize.xsmall,
         'icon-small': props.size === IconSize.small,
-        'icon-medium': props.size === IconSize.medium,
+        'icon-medium': props.size == null || props.size === IconSize.medium,
```

Comment:
what's the reason of this check?

Reply:
The component used to use prop defaults to always select medium as a default, I'm just following that same logic with this so if you don't pass a size it will select medium